### PR TITLE
Updates to Occurrence, Event, Taxon Cores plus related extensions and vocabularies

### PR DESCRIPTION
**Pull request by tucotuco**
Update to Occurrence Core to include latest species establishment term additions and numerous long-neglected amendments.

### CODE REVIEW

**Comment by peterdesmet** [comment]
Note: these terms should also be added to the distribution extension. Will see if I can do that tomorrow.

**Comment by tucotuco** [comment]
While you are at it, check the definitions of other terms. The Occurrence
Core was out of date back to 2015 in this respect.

**Comment by peterdesmet** [comment]
Oh, I don't have write access to your branch of course. No matter, I think it makes more sense to do this as two pull requests anyway. What do you mean "check the definition of other terms"? You mean of the distribution extension?

File: core/dwc_occurrence_2020-10-24.xml [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+    <property group='Occurrence' name='establishmentMeans' namespace='http://rs.tdwg.org/dwc/terms/' qualName='http://rs.tdwg.org/dwc/terms/establishmentMeans' dc:relation='http://rs.tdwg.org/dwc/terms/index.htm#establishmentMeans' dc:description='Statement about whether an organism or organisms have been introduced to a given place and time through the direct or indirect activity of modern humans. Recommended best practice is to use controlled value strings from the http://rs.tdwg.org/dwcem/ controlled vocabulary designated for use with this term. For details, refer to https://doi.org/10.3897/biss.3.38084' examples='&quot;native&quot;, &quot;nativeReintroduced&quot;, &quot;introduced&quot;, &quot;introducedAssistedColonisation&quot;, &quot;vagrant&quot;, &quot;uncertain&quot;' required='false'/>
```
Comment (peterdesmet) [inline core/dwc_occurrence_2020-10-24.xml:53]: I wonder if we should link to a thesaurus here, cf. `occurrenceStatus`

    thesaurus="http://rs.gbif.org/vocabulary/gbif/establishment_means.xml

Probably updated to `http://rs.gbif.org/vocabulary/gbif/establishment_means_2020-10-26.xml` (to be created)

File: core/dwc_occurrence_2020-10-24.xml [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+    <property group='Occurrence' name='degreeOfEstablishment' namespace='http://rs.tdwg.org/dwc/terms/' qualName='http://rs.tdwg.org/dwc/terms/degreeOfEstablishment' dc:relation='http://rs.tdwg.org/dwc/terms/index.htm#degreeOfEstablishment' dc:description='The degree to which an Organism survives, reproduces, and expands its range at the given place and time. Recommended best practice is to use controlled value strings from the http://rs.tdwg.org/dwcdoe/ controlled vocabulary designated for use with this term. For details, refer to https://doi.org/10.3897/biss.3.38084' examples='&quot;native&quot;, &quot;captive&quot;, &quot;cultivated&quot;, &quot;released&quot;, &quot;failing&quot;, &quot;casual&quot;, &quot;reproducing&quot;, &quot;established&quot;, &quot;colonising&quot;, &quot;invasive&quot;, &quot;widespreadInvasive&quot;' required='false'/>
```
Comment (peterdesmet) [inline core/dwc_occurrence_2020-10-24.xml:54]: Same here, I think we link to a thesaurus, either a new xml one hosted here at rs.gbif.org, or elsewhere.

**Comment by timrobertson100** [comment]
Thank you.

While you are exploring these, can you please also cast an eye over the event core and see if any of the terms common in both need to be addressed there?

We need to test the implications of this in a test IPT and production crawling in GBIF before we merge this. Can you please target the sandbox environment for the PR? A second PR will be used to move it from sandbox to production (by @MattBlissett or myself) and reviewed as an additional step.

**Comment by peterdesmet** [comment]
Now created a sister PR at https://github.com/gbif/rs.gbif.org/pull/47

**Comment by peterdesmet** [comment]
@timrobertson100 should we just do all this in the sandbox environment, ideally on the same branch? @tucotuco and I can coordinate

@timrobertson100 do you want us to create/copy the newly adopted controlled vocabularies for these terms in  http://rs.gbif.org/vocabulary/gbif/ or http://rs.gbif.org/vocabulary/dwc/ or reference an external link?

**Comment by timrobertson100** [comment]
> @timrobertson100 should we just do all this in the sandbox environment, ideally on the same branch? @tucotuco and I can coordinate

Yes, please. Feel free to do it in this PR by committing more to your branch.

> @timrobertson100 do you want us to create/copy the newly adopted controlled vocabularies for these terms in http://rs.gbif.org/vocabulary/gbif/ or http://rs.gbif.org/vocabulary/dwc/ or reference an external link?

You mean sandbox or production?  Everything needs to go to the sandbox first to allow testing.

**Comment by timrobertson100** [comment]
To ensure we don't block anything, suggest keep new work (chrono extension) in separate PR which can be merged immediately

**Comment by tucotuco** [comment]
I didn't realize I had this other pending work not in a pull request previously. Unintentional.

**Comment by timrobertson100** [comment]
Given this has no changes, should we close it and start afresh? I presume something has gone wrong with a merge.

**Comment by peterdesmet** [comment]
I will try to tackle this before the end of the year. I might start with a new PR.

**Comment by tucotuco** [comment]
I already have these done. Will commit and make a pull request momentarily.

**Comment by tucotuco** [comment]
Abandoning this pull request in favor of multiple smaller ones.

**Pull request closed without merging**